src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+    <form
+      data-qa="big"
+      class="form"
+      action="#"
+      method="post">
+      <img class="search-icon search-icon--big"
+        src="images/Search.svg"
+        alt="Big search icon"
+      >
```

Comment:
```suggestion
    <form
      data-qa="big"
      class="form"
      action="#"
      method="post"
    >
      <img 
        class="search-icon search-icon--big"
        src="images/Search.svg"
        alt="Big search icon"
      >
```
Don't leave comments and please pay attention to your code formatting
![image](https://user-images.githubusercontent.com/96409563/206662890-9da82504-2b71-4b45-9f98-81ec67d2aeb8.png)